Base/home/anon/.config/Tests.ini

```diff
@@ -1,7 +1,7 @@
 [Global]
 SkipDirectories=Kernel/Legacy
 SkipRegex=^ue-.*$
```

Comment:
This rule actually already covers the fuzzer, if you look at the logs you will see it's always skipped.

```
START  Kernel/Legacy/fuzz-syscalls (77/191)
PASS   Kernel/Legacy/fuzz-syscalls (0ms)
           Test:   fuzz-syscalls (skipped)
```

Reply:
Uhh, thaat is weird, because my other PR just timed out due to fuzz-syscalls o.O
https://dev.azure.com/SerenityOS/SerenityOS/_build/results?buildId=6416&view=logs&jobId=85a3409c-8a2a-5c59-a399-3144c1987667&j=85a3409c-8a2a-5c59-a399-3144c1987667&t=db5da47f-8506-5bb6-d73c-b7b743960fc8

Do you have an idea what might've happened there? I'm confused.

Reply:
Because clearly, *some* Legacy tests got executed:
![Bildschirmfoto_2021-12-25_13-26-39](https://user-images.githubusercontent.com/2690845/147384806-86d88e16-547d-4686-8881-9b93c387d2a6.png)

Reply:
They all say skipped at the end... I think the coloring just makes it confusing.

Reply:
Oof, yes, it was right in front of my, my fault.

Thanks! So I guess this was just a fluke then? It's kinda weird that it timed out for no reason.